### PR TITLE
DOCS-6791 language support

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -30,6 +30,8 @@ Click the Helpful Links item in the left navigation area for links to documentat
 
 Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
 
+Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.
```
Comment (kuntald) [inline modules/ROOT/pages/index.adoc]: @wwarner-salesforce , @mcampo :
this is good, but I think Language support should be a separate section/heading by itself. I don't know where or under which sub-heading. Any thoughts?

Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: I think the best option is to add a small section on the front page near the "About Content" section. I'll implement that now in this draft.

A second option is to add it to the "Create an Asset" page: https://docs.mulesoft.com/exchange/to-create-an-asset

Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Here's the separate section on the front page. Please let me know what you think.

**Review by kuntald — APPROVED** [review]

**Review by kbostrom — APPROVED** [review]
No comments; looks great!
I hope you feel better soon! There is something nasty going around, for sure.

**Review by mcampo — APPROVED** [review]

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -32,6 +32,10 @@ Exchange assets are accessible in Anypoint Studio (connectors, templates, and ex
 
 MuleSoft certifies its public content, and reviews and approves partner connectors. All private content is internal to an organization and not visible outside their organization. Customers are responsible for any private content they publish within their Exchange.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:34]: Looks good!

**Review by kbostrom — APPROVED** [review]
No changes

**Pull request merged**